soc/espressif/esp32c2/memory.h

```diff
@@ -31,7 +31,7 @@
 /* The offset between Dbus and Ibus.
  * Used to convert between 0x403xxxxx and 0x3fcxxxxx addresses.
  */
-#define IRAM_DRAM_OFFSET         0x6e0000
+#define IRAM_DRAM_OFFSET         (SRAM1_IRAM_START - SRAM1_DRAM_START)
```

Comment:
can we hardcode the DRAM - IRAM offset here? It should be constant to avoid issues when someone misconfigure the starting origins.